Source/Core/Duality/Drawing/Canvas.cs

```diff
@@ -394,6 +394,168 @@ public void DrawThickLine(float x, float y, float x2, float y2, float width)
 			this.DrawThickLine(x, y, 0, x2, y2, 0, width);
 		}
 
+		/// <summary>
+		/// Draws an image.
+		/// </summary>
+		public void DrawImage(ContentRef<Material> material, float x, float y, float z, float w, float h)
```

Comment:
This overload (and all others based on `Material` references) can just forward to the `BatchInfo` one and provide [material.Info](https://github.com/AdamsLair/duality/blob/b31fbab9c85bf3e018dd9025f510f0ae25a19784/Source/Core/Duality/Resources/Material.cs#L85) as a parameter.